### ISSUE TITLE
Fix/enable caching of alchemical tests requiring simulation data for test_alchemy

### ISSUE THREAD

**Issue by jchodera** [choderalab/openmmtools#171]
It looks like the caching of simulation data by `test_alchemy.py` is both disabled and broken. We may want to fix this and re-enable it at some point.

**Comment by jchodera** [comment]
Yes, I just disabled them because I couldn't figure out how to get them to work.

**Comment by andrrizzi** [comment]
Ok! I'll take a look at it now.
